### DISCUSSION TITLE
Are you sent to me info about your project?

### DISCUSSION THREAD

**Discussion by bikevit2008** [projectdiscovery/nuclei#4776] [General]
<?php echo "Nuclei - Open source project (github.com/projectdiscovery/nuclei)"; phpinfo(); ?>

File uploaded to my site onplay.me

:)

**Comment by olearycrew** [comment]
I'm sorry I don't understand what you're asking.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
